testcontainers/kafka.py

```diff
@@ -11,6 +11,16 @@
 
 
 class KafkaContainer(DockerContainer):
+    """
+    Kafka container.
+
+    Example
+    -------
+    ::
```

Comment:
Would you be up for updating all the examples to use the `doctest` directive (see [here](https://github.com/testcontainers/testcontainers-python/pull/251/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fL47-R47) for an example)? That will ensure the documentation always stays in sync with the code.

Reply:
Wow, I didn't know the doctest directive, it is amazing. :D
I am going to implement this at every test!

Reply:
@tillahoffmann added the doctest directives :)

Reply:
I changed also the sphinx tests to a seperate job in the workflow.